lib/home/model/RequestsController.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:Radar/home/model/Request.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_secure_storage/flutter_secure_storage.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:location/location.dart';
+import 'package:nearby_connections/nearby_connections.dart';
+
+class RequestsController extends ChangeNotifier {
+  final FlutterSecureStorage _secureStorage;
+  final Nearby _nearby = Nearby();
+  final Location _location = Location();
+  final Strategy _strategy = Strategy.P2P_CLUSTER;
+  final List<Request> requests = [];
```

Comment:
Error handling is missing. But we could probably leave this for now

Reply:
Ya i was planning to do handle errors when the whole request functionality is complete or somewhat stable as i was still discovering how to implement the library and hence there were many changes.